validate_email/mx_check.py

```diff
@@ -78,7 +79,7 @@ def _smtp_ehlo_tls(smtp: SMTP, helo_host: str):
     unavailable.
     """
     code, message = smtp.ehlo(name=helo_host)
-    if code >= 300:
+    if code >= 400:
```

Comment:
Do you handle 3xx responses here? Not sure if it doesn't need to be an error, but since they're not handled, I'd keep the original here.

Reply:
Responses 2xx and 3xx generally indicate success, only 4xx and 5xx indicate failure. Actually at this point in the communication, I don't think if any 3xx response is even possible, but if there is one, it would not indicate an error.

Reply:
`3yz (Positive Intermediate Reply): The command has been accepted, but the requested action is being held in abeyance, pending receipt of further information.`

Alright, I'll resolve this part then.